install.sh

```diff
@@ -368,6 +372,9 @@ fi
 # We append customary rocm path; if user provides custom rocm path in ${path}, our
 # hard-coded path has lesser priority
 export PATH=${PATH}:/opt/rocm/bin
+if ! [ -z ${ROCM_PATH+x} ]; then
+    export PATH=${PATH}:${ROCM_PATH}/bin
```

Comment:
This is appending the ROCM_PATH to the end, it is useless as /opt/rocm/ is supposed to be available as a software link and always exists.

Reply:
The symlink is needed only when user is using frameworks like pytorch/Caffe2 or any benchmark framework. It helps to keep an uniform interface when single/multiple ROCm versions are installed on a machine. 
In the above scenario while compiling the rocFFT it is required to point to the exact path of ROCm so that we pick the correct version of compiler/hcc binary.

Reply:
As already mentioned, symlink doesn't always exist.